lib/enums/symptoms.dart

```diff
@@ -0,0 +1,12 @@
+enum Symptom {
+  Fever,
+  Cough,
+  ShortOfBreath,
+  FeelingIll,
+  Headache,
+  BodyAches,
+  OddTaste,
+  OddSmell,
+  SneezingRunnyNose,
+  SoreThroat
```

Comment:
```suggestion
  SoreThroat,
  NauseaVomiting,
  Diarrhea
```

Reply:
Separated Sneezing from RunnyNose
Added NauseaVomiting (together) and Diarrhea, as these symptoms have also been recognized